Cargo.lock

```diff
@@ -1250,6 +1250,12 @@ version = "1.2.2"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "c3ac9f8b63eca6fd385229b3675f6cc0dc5c8a5c8a54a59d4f52ffd670d87b0c"
 
+[[package]]
```

Comment:
Moving this tool into workspace adds a lot of dependencies into `Cargo.lock`. Plus it now requires compilation of this crate during `cargo test`.

I still prefer not to include it into workspace. Can we have it in a separate repository? Maybe in `fuel-utils`?

Reply:
That's where I feel it should live long-term, but I find that I'm constantly making changes to the algorithm code when I'm working on this and it would be nice to have them in the same place.